apps/condo/domains/notification/helpers/messageTools.js

```diff
@@ -0,0 +1,76 @@
+const { get, isEmpty } = require('lodash')
+
+const { MESSAGE_META } = require('@condo/domains/notification/constants/constants')
+const { WRONG_MESSAGE_TYPE_PROVIDED_ERROR } = require('@condo/domains/notification/constants/errors')
+
+/**
+ * Fills payload using data according to message meta settings by selected notification type.
+ * Can handle nested paths if provided.
+ * @param messageType MESSAGE_META key
+ * @param data { key:value, ...}
+ * @param nestedPath ['String', ...]
+ * @returns {*}
+ */
+const fillDataByMessageTypeMeta = (messageType, data, nestedPath = []) => {
```

Comment:
Move it to the resident domain. You don't need it inside this domain. It looks like a local util